### PR TITLE
fix: cleanup the concluded engagement experiments

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Cleaned up the engagement loop experiments
- Companion => Lost => Removal
- Upvote => Won => Made default

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/hooks/useOnPostClick.ts [hunk 1]
```diff
@@ -67,10 +65,6 @@ export default function useOnPostClick({
                 null,
                 optional?.parent_id,
               ).extra,
-              feedback:
-                isFeedbackEnabled && post.type === PostType.Article
-                  ? true
-                  : undefined,
```
Comment (rebelchris) [inline packages/shared/src/hooks/useOnPostClick.ts:73]: @idoshamun I need to wait for [this](https://dailydotdev.slack.com/archives/C04UN615UAF/p1698388838610749) to resolve right?

File: packages/shared/src/hooks/feed/useTrackImpression.ts [hunk 2]
```diff
@@ -42,10 +39,6 @@ export default function useTrackImpression(
               ...feedAnalyticsExtra(feedName, ranking, {
                 scroll_y: window.scrollY,
               }).extra,
-              feedback:
-                isFeedbackEnabled && item.post.type === PostType.Article
-                  ? true
-                  : undefined,
```
Comment (rebelchris) [inline packages/shared/src/hooks/feed/useTrackImpression.ts:48]: @idoshamun Same here, waiting for [this](https://dailydotdev.slack.com/archives/C04UN615UAF/p1698388838610749).

**Review by capJavert — APPROVED** [review]
Nice cleanup!

**Review by sshanzel — APPROVED** [review]
It is always a good day to see code getting deleted :shipit:

File: packages/shared/src/hooks/useOnPostClick.ts (hunk 1 above)
Comment (idoshamun) [reply packages/shared/src/hooks/useOnPostClick.ts:73]: Yes for both comments, either we keep it for now or we keep this PR pending. Up to you

File: packages/shared/src/hooks/feed/useTrackImpression.ts [hunk 3]
```diff
@@ -46,6 +46,7 @@ export default function useTrackImpression(
                 isFeedbackEnabled && item.post.type === PostType.Article
```
Comment (idoshamun) [inline packages/shared/src/hooks/feed/useTrackImpression.ts:46]: isFeedbackEnabled should always return true right?

**Review by idoshamun — APPROVED** [review]
Only one question regarding is feedback enabled, I think it should be true always

File: packages/shared/src/hooks/feed/useTrackImpression.ts (hunk 3 above)
Comment (rebelchris) [reply packages/shared/src/hooks/feed/useTrackImpression.ts:46]: Modified, let me know what you think

**Review by idoshamun — APPROVED** [review]

**Pull request merged**